docs/source/developers/contributing.rst

```diff
@@ -304,3 +304,40 @@ to your branch, which they sometimes do to help move a pull request along.
 In addition, the GitHub PR "suggestion" feature can also add commits to
 your branch, so it is possible that your local copy of your branch is missing
 some additions.
+
+Guidance for specific features
+==============================
+
+From time to time the community has discussions on specific types of features
+and improvements that they expect to support.  This section outlines decisions
+that have been made in this regard.
+
+Endianess
++++++++++
+Arrow is primarily a little endian format there has been some effort to
```

Comment:
I wouldn't say that. Arrow lets you specify the endianness. But many computer systems these days (most?) are little-endian and most Arrow-based applications probably run on little-endian systems.

(note "systems": e.g. PPC CPUs can be configured in both LE and BE modes, but under Linux they use LE)

Reply:
rephrased to account for popularity of endiannes.